---
layout: page
title: Digital CV
---

> **Attention!!**  Please notice that GutMeta is currently unavailable. Will release a new notice if my previous colleaguse fix the issue and bring back the website.


## Skills

<table>
  <tbody>
    <tr>
      <td>OS</td>
      <td>MAC, Linux, Windows</td>
    </tr>
    <tr>
      <td>Programming Languages</td>
      <td>Skilled Java, C#, JavaScript, Typescript, Python3, Ruby; <br>Medium C++;
      </td>
    </tr>
    <tr>
      <td>Web Frameworks</td>
      <td>Spring MVC, Spring Boot, Ruby On Rails, Vue.Js, Angular.Js</td>
    </tr>
    <tr>
      <td>Relational and NoSQL Database</td>
      <td>Microsoft SQL, Mysql, Postgresql, Cassandra</td>
    </tr>
    <tr>
      <td>Data Mining Skills</td>
      <td>Spark, Python(pandas)</td>
    </tr>
     <tr>
      <td>Developer's tools</td>
      <td>VSCode, Ecllipse, Postman, NetBeans, Git</td>
    </tr>
  </tbody>
</table>


## Education

|time|institution|
|--|--|
|2022 - Now | Simon Fraser University <br>Master in Professional Computing Science, big data|
|2014 - 2018 | City University of Hong Kong <br>Bachelor of Science(Computer Science)|

<!-- |2011 - 2014 | Shang Hai Jincai High School |
|2007 - 2011 | Shang Hai Jincai North Secondary School| -->

<hr>

## Main Research Projects


### 1. GutMeta

- **intro**: GutMeta is a one-stop platform for human gut metagenomics research. 
- **responsibilities**:
    - Designed the platform in collaboration with other Phd students,
    - Designed the back-end database;
    - Implemented the webframework using Ruby On Rails, Vue.js, and PostgreSQL;
    - Developed all the online visualization modules using Oviz.

- **result**: The proposal is now under revision at Microbiome. The platform  is available at [gutmeta.deepomics.org](https://gutmeta.deepomics.org/visualizer/analysis/meta-overview).

### 2. CNAHap

- **intro**: CNAHap is a germline haplotyping tool using tumor allele-specific copy number alteration. For this project, I developed the online visualization modules, including a novel compound diagram _Phased:On Genes_ and a Circos Overview chart, using the Oviz-Bio platform.
- **result**: One paper available at [bioRxiv](https://www.biorxiv.org/content/10.1101/2021.03.27.437314v1). The visualizations are available on bio-oviz, you can check [this post]({% post_url 2021-11-03-use-bio-oviz %}) for the guidelines.
<!-- - Tan B, Chen L, Jia W, Wang Y, Li H, Li S. CNAHap: a germline haplotyping method using tumor allele-specific copy number alteration. bioRxiv. 2021 Jan 1. -->

<hr>

## Competition
### CombCure â€“ an intelligent cancer clinical platform
_(Mar 2021 - Now)_

> **Intro**: CombCure is a web-based platform designed for doctors. CombCure analyzes the genome sequnces of the patient (obtained from third-party sequencing organization), searches the curated database for the most appropriate medicines using our AI algorithms, and generates a report for doctors with visualized analysis results and medicine recommendation.

#### **Responsibilities**
- Act as the CTO for the project.
- Draft and amend the business proposal; present the project to the council.
- Devise the back-end system; give tutorials to intern students and direct their work on platform development; supervise the development progress.

#### **Results**
- The [project](https://www.cityu.edu.hk/hktech300/start-ups/seed-fund-teams/combcure) has won the seed fund for the first cohort in [HKTech300](https://www.cityu.edu.hk/hktech300/home).
- Beta version named Gene-App established, with all core functions implemented, including user management, analysis task submission, and report generation.

<hr>

## Work Experiences
**CAE Inc** 
<br>_(May 2023 - Dec 2023)_
- Worked as a **full-stack developer** in an Agile Scrum team for CAE Pelesys.
- Maintained the aviation e-learning system using C#, .Net, and Vue.js by adding features and fixing bugs to ensure the quality of the system and improve customer satisfaction
- Analyze UI loading performance issue by tracing network request latencies with Edge software developers
- Collaborated with peers to implement the UI of the notification system with Vue.js and Typescript, successfully delivering it as the Minimum Viable Product(MVP)
- Implemented the background process for notifying certificate status updates using .Net, C#, and RESTful API calls, successfully delivering the requested feature and ensuring customers satisfaction

**City University of Hong Kong, Dept of Computer Science** 
<br>_(Oct 2020 - Sep 2022)_
- Worked as a **research assistant** for Professor LI Shuaicheng.
- Designed database system for GutMeta website.
- Developed and maintain the websites for academic research achievements using Ruby On Rails, Postgresql, Vue.JS and Oviz.
- Developed and maintain data visualization modules on the web platform using Oviz.
- Maintained the Oviz framework by adding features and adding documentation.
> [Oviz](https://oviz.org) is a visualization framework created by a previous RA of Prof. Li. Oviz has its own template language to describe either an SVG or a canvas chart and adopts Typescript for other functions such as data processing and interaction.

**HID Global Cid Ltd.**
<br>_(May 2019 - Sep 2020)_
- Worked as a **junior solution developer**
- Developed back-end systems of the digital-security-document web applications, using Java, Spring Framework, and MSSQL.
- Maintained the e-passport systems for Philippine and Nigeria.
- Prepared report templates using Jaspersoft studio.

**L&E Management** _(worked for HID)_
<br>_(Jul 2018 - Apr 2019)_
- Developed back-end systems of the digital-security-document web applications.
- Optimized SQL queries for Philippine projects.
> At this time, I was working at HID Global, then transferred HID Global in May 2019.

<hr>

## Publications

1. Tan B, Chen L, Jia W, **Wang Y**, Li H, Li S. CNAHap: a germline haplotyping method using tumor allele-specific copy number alteration. bioRxiv. 2021 Jan 1.
2. He J, Luo J, Liu L, Chen L, Xu X, **Wang Y**, Wei B, Ju C, Wang X, Huang L, Zeng W, Miao X, Sang L, Huang D, Pan G, Peng G, Chen Z, Zhao Z, Yang C, Cui W, Li S. Over-shedding of donor-derived cell-free DNA at immune-related regions into plasma of lung transplant recipient. (_in production at CTM_)
3. Jiang Y\*, **Wang Y\***,  Che L*, Li, S. GutMeta, a one-stop platform for human gut whole-genome shotgun metagenomic sequencing data analysis (_proposal submitted to NAR_)
4. Wang S\*, Wang M\*, Chen L, Yu Y, Pan G, **Wang Y**; Tan B, Li S. SpecHLA: haplotyping entire HLA genes from sequencing data (_proposal submitted to Cell Systems_)
5. Jiang Y\*, Wang S, **Wang Y**, Zhang X, Li, S. A framework to trace microbial engraftment at the strain level during fecal microbiota transplantation (_under publication at BMC_) 
